universal-login-wallet/src/ui/react/Home/PrivacyPolicy.tsx

```diff
@@ -0,0 +1,81 @@
+/* eslint-disable */
+import React from 'react';
+
+export const PrivacyPolicy = () => {
+  return (
+    <div className="privacy-description">
```

Comment:
When we start working on multi language support, perhaps it would be better if we use markdown files for each document like that.